scripts/add-family-discussions.py

```
@@ -0,0 +1,102 @@
+    "Smurf": ["smurf"],
+    "Soft Twilight": ["Soft_Twilight_1.0"],
+    "Totoro-ify": ["Totoro-ify"],
+    "W I D E": ["W_I_D_E"],
+    "2024 XMR SimpleX": ["2024XMRSimpleX"],
+    "3a Rule": ["3aRule"],
+}
+
+headers = {"Authorization": f"Bearer {TOKEN}", "Content-Type": "application/json", "Accept": "application/vnd.github+json"}
+
+# Get existing
+resp = requests.post(API_URL, headers=headers, json={
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)